ruff.schema.json

```diff
@@ -4368,18 +4340,17 @@
         {
           "description": "Ban imports that extend into the parent module or beyond.",
           "type": "string",
-          "enum": [
-            "parents"
-          ]
+          "const": "parents"
         },
         {
           "description": "Ban all relative imports.",
           "type": "string",
-          "enum": [
-            "all"
-          ]
+          "const": "all"
         }
       ]
+    },
+    "string": {
+      "type": "string"
```

Comment:
This is related to https://github.com/astral-sh/ruff/pull/20942/files#r2440788641.